core/src/main/resources/io/questdb/site/conf/server.conf

```diff
@@ -329,6 +332,9 @@ query.timeout.sec=60
 # size of InsertModel pool in SqlParser
 #cairo.sql.insert.model.pool.capacity=64
 
+# batch size of non-atomic inserts for INSERT INTO SELECT statements
+#cairo.sql.insert.model.batch.size=8192
```

Comment:
This looks like very low number.

We typically not on the best code path when we're dealing with small transactions. 8k rows at a time falls into category of small. We should try to find optimal number. I will do some testing.

Reply:
1000000